wemake_python_styleguide/visitors/tokenize/conditions.py

```diff
@@ -57,11 +57,36 @@ def visit_name(self, token: tokenize.TokenInfo) -> None:
         """
         self._check_implicit_elif(token)
 
+    def _check_else_belongs_to_if(self, start_index: int) -> bool:
```

Comment:
We use `_check` prefix for methods that work with nodes. Let's rename it to `_does_belong_to_if`